src/main/scala/dsptools/counters/Counters.scala

```diff
@@ -164,11 +165,13 @@ abstract class Counter(countParams: CountParams) extends Module {
       //    else Mux(wrap,wrapTo,nextInSeq)
     }
     else {
-      if(wrap == Bool(true)) {
-        wrapTo
-      } else {
-        nextInSeq
+      val nextThing = Wire(Bool())
```

Comment:
Ok, sorry. Again didn't pay attention to Chick's changes. My original code was 

```
val up = {
    if (countParams.incMax == 1 || (countParams.wrapCtrl == External && countParams.customWrap))
      (io.out + inc).shorten(countParams.countMax)
    else upCustom
  }
```

Which is purely compile time.